src/services/expenses.services.js

```diff
@@ -0,0 +1,135 @@
+/* eslint-disable space-before-function-paren */
+'use strict';
+
+const { sequelize } = require('../database/db');
+const { DataTypes } = require('sequelize');
+
+const Expense = sequelize.define('Expense', {
+  userId: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+  },
+  spentAt: {
+    type: DataTypes.DATE,
+    allowNull: false,
+  },
+  title: {
+    type: DataTypes.TEXT,
+    allowNull: false,
+  },
+  amount: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+  },
+  category: {
+    type: DataTypes.TEXT,
+    allowNull: false,
+  },
+  note: {
+    type: DataTypes.TEXT,
+    allowNull: true,
+  },
+}, {
+  tableName: 'expenses',
+});
+
+const normalize = ({
+  id,
+  userId,
+  spentAt,
+  title,
+  amount,
+  category,
+  note,
+}) => {
+  return {
+    id,
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  };
+};
```

Comment:
I really don't understand what's the purpose of this function? it is take and return the same type of data. please explain me